.restyled.yaml

```diff
@@ -74,6 +74,7 @@ exclude:
     - "examples/chef/sample_app_util/test_files/*.yaml"
     - "examples/chef/zzz_generated/**/*"
     - "examples/platform/nxp/k32w/k32w0/scripts/demo_generated_certs/**/*"
+    - "examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/zap-generated/*" # zap-generated files
```

Comment:
can we place generated files under /zzz_generated instead of under examples?

It would be ideal to not mix generated files with non-genreated files. We should also probably prefer compile-time generation instead of checking in generated files. Was that path investigated?

Reply:
Hey Andrei. We have an example XCode project and XCode needs these files to be in the same folder. This is similar to how it is done for the Darwin Matter framework, under src/darwin/Framework/CHIP/zap-generated. 

I have previous commit which pre-generates some files in this directory. So I created a github issue, to be completed in a separate PR, to switch to compile-time generation instead of checking in generated files for examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/zap-generated/* 

https://github.com/project-chip/connectedhomeip/issues/32635

I didn't realize compile-time generation was an option since all the example files I looked at were pre-generated using “./scripts/tools/zap_regen_all.py”. 

What's the advantage of compile-time generation vs. pre-generation?

Reply:
That makes sense. I will address that as part of this Github issue in another PR: https://github.com/project-chip/connectedhomeip/issues/32635